source/administration/changelog.md

```diff
@@ -515,6 +521,11 @@ Many thanks to all our contributors. In alphabetical order:
 
 ### Notes on Patch Release
 
+ - **v3.6.7, released 2017-04-27**
+   - Fixed a number of security issues, and [upgrading](http://docs.mattermost.com/administration/upgrade.html) is highly recommended. Details will be posted on our [security updates page](https://about.mattermost.com/security-updates/) 14 days after release as per the [Mattermost Responsible Disclosure Policy](https://www.mattermost.org/responsible-disclosure-policy/)
+     - Note: The **System Console > Configuration > [Site URL](../../administration/config-settings.html#site-url)** field is now mandatory. Set the Site URL in the System Console, or in the `gitlab.rb` file if you are using GitLab Mattermost.
+ - **v3.6.6, released 2017-04-13**
```

Comment:
we also fixed an issue where DM list didn't always properly update on LHS
also upgraded mysql driver for better performance

^0/5 on whether we should include in the changelog for 3.6.6 or not